Packs/FalconX/Integrations/FalconX/FalconX.py

```diff
@@ -0,0 +1,415 @@
+from typing import Dict, Callable, Tuple, Any
+
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+import json
+import requests
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+URL = "https://api.crowdstrike.com/"
```

Comment:
Please put it as a defaultvalue in the parameter, and avoid using as global, all should be handled in main.